packages/fast-components-styles-msft/src/context-menu-item/index.ts

```diff
@@ -13,52 +10,62 @@ import {
     neutralForegroundHover,
     neutralForegroundRest,
 } from "../utilities/color";
-import { applyFocusVisible } from "@microsoft/fast-jss-utilities";
-import { applyTypeRampConfig } from "../utilities/typography";
-import { toPx } from "@microsoft/fast-jss-utilities";
+import { applyFocusVisible, toPx } from "@microsoft/fast-jss-utilities";
+import { applyCornerRadius, applyFocusPlaceholderBorder } from "../utilities/border";
+import { applyDisabledState } from "../utilities/disabled";
+import { scaleApplyTypeRampConfigWithDensity } from "../utilities/typography";
 
-const styles: ComponentStyles<ContextMenuItemClassNameContract, DesignSystem> = {
-    contextMenuItem: {
-        listStyleType: "none",
-        height: density(32),
-        display: "grid",
-        boxSizing: "border-box",
-        gridTemplateColumns: "40px auto 40px",
-        gridTemplateRows: "auto",
-        alignItems: "center",
-        padding: "0",
-        margin: "0 4px",
-        color: neutralForegroundRest,
-        whiteSpace: "nowrap",
-        overflow: "hidden",
-        cursor: "default",
-        transition: "all 0.2s ease-in-out",
-        ...applyTypeRampConfig("t7"),
-        borderRadius: (config: DesignSystem): string => {
-            const designSystem: DesignSystem = withDesignSystemDefaults(config);
+const styles: ComponentStyles<ContextMenuItemClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<ContextMenuItemClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const padding: number =
+        horizontalSpacingNumber(-2)(designSystem) +
+        16 +

```

Comment:
should we define this number as a const so its purpose is understood? It seems like an arbitrary number sandwiched between these two.

Reply:
Sure. It represents the icon width. I considered pulling that out, but we don't have a story around scaling icons. I'll keep it local to this file, and maybe someday we'll address whether those scale with density or not.